2024-08-13 17:20:34,103 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-08-13 17:20:34,103 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Configure stats pid to 26544
2024-08-13 17:20:34,103 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from C:\Users\harsh\.config\wandb\settings
2024-08-13 17:20:34,104 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\settings
2024-08-13 17:20:34,104 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-08-13 17:20:34,104 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-13 17:20:34,105 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-08-13 17:20:34,106 INFO    MainThread:26544 [wandb_init.py:_log_setup():524] Logging user logs to e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\run-20240813_172034-d4e2p39g\logs\debug.log
2024-08-13 17:20:34,106 INFO    MainThread:26544 [wandb_init.py:_log_setup():525] Logging internal logs to e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\run-20240813_172034-d4e2p39g\logs\debug-internal.log
2024-08-13 17:20:34,106 INFO    MainThread:26544 [wandb_init.py:init():564] calling init triggers
2024-08-13 17:20:34,106 INFO    MainThread:26544 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-08-13 17:20:34,107 INFO    MainThread:26544 [wandb_init.py:init():589] re-initializing run, found existing run on stack: hpcwvdts
2024-08-13 17:20:34,111 INFO    MainThread:26544 [wandb_run.py:_finish():1962] finishing run harshmp/truck-delay-classification/hpcwvdts
2024-08-13 17:20:34,116 INFO    MainThread:26544 [jupyter.py:save_history():445] not saving jupyter history
2024-08-13 17:20:34,117 INFO    MainThread:26544 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-08-13 17:20:34,117 INFO    MainThread:26544 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-08-13 17:20:34,117 INFO    MainThread:26544 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-08-13 17:20:34,118 INFO    MainThread:26544 [wandb_run.py:_restore():2185] restore
2024-08-13 17:20:34,118 INFO    MainThread:26544 [wandb_run.py:_restore():2191] restore done
2024-08-13 17:20:39,407 INFO    MainThread:26544 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-08-13 17:20:39,408 INFO    MainThread:26544 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-08-13 17:20:39,417 INFO    MainThread:26544 [wandb_run.py:_footer_sync_info():3796] logging synced files
2024-08-13 17:20:39,464 INFO    MainThread:26544 [wandb_init.py:init():614] starting backend
2024-08-13 17:20:39,464 INFO    MainThread:26544 [wandb_init.py:init():618] setting up manager
2024-08-13 17:20:39,468 INFO    MainThread:26544 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-08-13 17:20:39,470 INFO    MainThread:26544 [wandb_init.py:init():624] backend started and connected
2024-08-13 17:20:39,490 INFO    MainThread:26544 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-08-13 17:20:39,491 INFO    MainThread:26544 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-08-13 17:20:39,491 INFO    MainThread:26544 [wandb_init.py:init():716] updated telemetry
2024-08-13 17:20:39,608 INFO    MainThread:26544 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-08-13 17:20:40,821 INFO    MainThread:26544 [wandb_run.py:_on_init():2254] communicating current version
2024-08-13 17:20:40,940 INFO    MainThread:26544 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.17.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-08-13 17:20:40,940 INFO    MainThread:26544 [wandb_init.py:init():800] starting run threads in backend
2024-08-13 17:20:41,974 INFO    MainThread:26544 [wandb_run.py:_console_start():2233] atexit reg
2024-08-13 17:20:41,974 INFO    MainThread:26544 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-08-13 17:20:41,975 INFO    MainThread:26544 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-08-13 17:20:41,975 INFO    MainThread:26544 [wandb_run.py:_redirect():2178] Redirects installed.
2024-08-13 17:20:41,977 INFO    MainThread:26544 [wandb_init.py:init():841] run started, returning control to user process
2024-08-13 17:22:35,353 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-08-13 17:22:35,354 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Configure stats pid to 26544
2024-08-13 17:22:35,355 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from C:\Users\harsh\.config\wandb\settings
2024-08-13 17:22:35,355 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\settings
2024-08-13 17:22:35,356 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-08-13 17:22:35,356 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-13 17:22:35,358 INFO    MainThread:26544 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-08-13 17:22:35,360 INFO    MainThread:26544 [wandb_init.py:_log_setup():524] Logging user logs to e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\run-20240813_172235-53dw67a2\logs\debug.log
2024-08-13 17:22:35,362 INFO    MainThread:26544 [wandb_init.py:_log_setup():525] Logging internal logs to e:\Artificial Intelligence\Projects\Predictive-Truck-Delay-Management-in-Logistics\wandb\run-20240813_172235-53dw67a2\logs\debug-internal.log
2024-08-13 17:22:35,362 INFO    MainThread:26544 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002ABD81AD120>
2024-08-13 17:22:35,363 INFO    MainThread:26544 [wandb_init.py:init():564] calling init triggers
2024-08-13 17:22:35,363 INFO    MainThread:26544 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-08-13 17:22:35,364 INFO    MainThread:26544 [wandb_init.py:init():589] re-initializing run, found existing run on stack: d4e2p39g
2024-08-13 17:22:35,369 INFO    MainThread:26544 [wandb_run.py:_finish():1962] finishing run harshmp/truck-delay-classification/d4e2p39g
2024-08-13 17:22:35,370 INFO    MainThread:26544 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-08-13 17:22:35,370 INFO    MainThread:26544 [wandb_run.py:_restore():2185] restore
2024-08-13 17:22:35,371 INFO    MainThread:26544 [wandb_run.py:_restore():2191] restore done
2024-08-13 17:22:39,197 INFO    MainThread:26544 [wandb_run.py:_footer_history_summary_info():3837] rendering history
2024-08-13 17:22:39,197 INFO    MainThread:26544 [wandb_run.py:_footer_history_summary_info():3869] rendering summary
2024-08-13 17:22:39,204 INFO    MainThread:26544 [wandb_run.py:_footer_sync_info():3796] logging synced files
